src/main/resources/i18n/messages.properties

```diff
@@ -341,7 +341,44 @@ project.members.add.error=An error occurred while adding the user to the project
 project.members.add.required=User and project role must be selected before submitting.
 project.members.remove.modal.confirm=You are about to remove <strong>{0}</strong> from the project.
 
-
+# ========================================================================================== #
+# ShareSamples Component                                                                     #
+# ========================================================================================== #
+ShareNoSamples.description=You have no samples selected to share, please go back to the project samples page and select some samples
+ShareNoSamples.link=Go to Project Samples Page
+ShareSamples.title=Share Samples with Another Project
+ShareSamples.projects=Select a project to share samples with
+ShareSamples.ready=These samples are ready to copy
+ShareSamples.exists=Samples that exist in the target project and will not be copied
+ShareSamples.avatar.unlocked=This same is modifiable in this project.
+ShareSamples.avatar.locked=This same is not modifiable in this project.
+ShareSamples.remove=Remove this sample from being copied.
+ShareSamplesList.title=Review samples to share
+ShareSamples.no-samples.message=All samples exist in the target project
+ShareSamples.no-samples.description=Since these samples exists, there is no reason to re-share them.
+ShareSamples.some-samples.message={0} samples exist in the target project and will not be re-shared.
+ShareSamples.checkbox.remove=Remove samples from current project (move samples)
+ShareSamples.checkbox.lock=Prevent modification of samples in target project (only when copying samples)
+ShareButton.button=Share Samples
+server.ShareSamples.move-error=There was an error moving these samples to {0}.  If the problem persists, please contact a system administrator.
+server.ShareSamples.copy-error=There was an error copying these samples to {0}.  If the problem persists, please contact a system administrator.
+ShareError.reset=Start Over
+ShareError.return=Return to Project's Samples
```

Comment:
Change to `Return to Project Samples page`

Reply:
Updatd in 3598392